### DISCUSSION TITLE
part-ii/2020-85/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#314] [Question comments]
Part II 2020 Linear Analysis: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2020-85/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(b) The closure of an equicontinuous set is equicontinuous, so wlog $C(K)$ is a countable union of closed equicontinuous sets. They can't all have empty interior by Baire Category Theorem, so the closed unit ball is equicontinuous. By Arzela Ascoli, it's compact, so $C(K)$ is finite dimensional, hence $K$ is finite (ES3Q2).

(c) $x_k$ is equibounded and equicontinuous using that $F$ is bounded. The Arzela Ascoli solution is shown to solve the DE using uniform continuity arguments. 

</details>
